### PR TITLE
🚜 Delete unneeded fields

### PR DESCRIPTION
**Pull request by drewbrew**
As suggested by @mtrythall: https://github.com/djangocon/2024.djangocon.us/pull/37#issuecomment-2218860922

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -32,11 +32,9 @@ class FrontmatterModel(BaseModel):
     """
 
     date: pydatetime.datetime | None
-    layout: str
     permalink: str | None
     redirect_from: list[str] | None
     redirect_to: str | None  # via the jekyll-redirect-from plugin
-    sitemap: bool | None
```
Comment (jefftriplett) [inline main.py:39]: The rest looked fine, but will data leak into our sitemap file without this? Previously, this was an issue, which is what led to it.

**Review by jefftriplett — APPROVED** [review]
I left one question, but we can always add it back.

File: main.py (hunk 1 above)
Comment (drewbrew) [reply main.py:39]: that's a question for @mtrythall

Comment (jefftriplett) [reply main.py:39]: I remember @mtrythall saying he added support for sitemaps a month or two ago. If the new system has a better way, I just want to make sure we aren't leaking info to search engines before we are ready. (This was previously an issue we fixed)

Comment (drewbrew) [reply main.py:39]: Added it back in just in case

**Pull request merged**